spk/couchpotatoserver/Makefile

```diff
@@ -1,31 +1,32 @@
 SPK_NAME = couchpotatoserver
 SPK_VERS = $(shell date +%Y%m%d)
-SPK_REV = 6
+SPK_REV = 7
 SPK_ICON = src/couchpotatoserver.png
-DSM_UI_DIR = app
 
 DEPENDS =
 SPK_DEPENDS = "python>2.7.3-3:git"
```

Comment:
I would highly doubt it, but maybe we can ask the people reporting problems now to test the new packages? (since we already have their attention)
This way we get feedback and know for sure.